aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -319,6 +319,22 @@ SNI support requires:
 * Running on target framework `netcoreapp2.1` or later. On `net461` or later, the callback is invoked but the `name` is always `null`. The `name` is also `null` if the client doesn't provide the host name parameter in the TLS handshake.
 * All websites run on the same Kestrel instance. Kestrel doesn't support sharing an IP address and port across multiple instances without a reverse proxy.
 
+## SSL/TLS Protocols
+
+SSL Protocols are protocols used for encrypting and decrypting traffic between two peers, traditionally a client and a server.
+
+```c#
+webBuilder.ConfigureKestrel(serverOptions =>
+{
+    serverOptions.ConfigureHttpsDefaults(listenOptions =>
+    {
+        listenOptions.SslProtocols = SslProtocols.Tls13;
+    });
+});
+```
+
+By default, Kestrel will use the Operating System defaults to choose the best protocol to use. Unless you have a specific reason to select a protocol, you use this default.
```

Comment:
```suggestion
By default, Kestrel uses the operating system defaults to choose the best protocol. Unless you have a specific reason to select a protocol, use the default.
```

Reply:
How about mentioning `SslProtocols.None` is the default value?

```suggestion
The default value, `SslProtocols.None`, causes Kestrel to use the operating system defaults to choose the best protocol. Unless you have a specific reason to select a protocol, use the default.
```